src/timeMachine/utils/queryBuilder.ts

```diff
@@ -165,6 +165,12 @@ const convertTagsToFluxFunctionString = function convertTagsToFluxFunctionString
   }
 
   if (tag.aggregateFunctionType === 'group') {
+    // if group is selected, but there are no values, don't return anything.
+    // same behavior as when 'filter' is selected but no values are chosen
+    if (!tag.values.length) {
+      return ''
+    }
```

Comment:
what do you think about adding a test for this case? there's already a test file written for this file.

Reply:
added